src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Add a metric for total paused duration in GC
```

Comment:
for .NET 7 and grater?

Same for documentation.

Reply:
Good point. I modified CHANGELOG and PR title.

In documentation I have a note just like other metrics that are only for .NET 7 or later.